src/mpol/plot.py

```diff
@@ -38,6 +38,65 @@ def get_image_cmap_norm(image, stretch='power', gamma=1.0, asinh_a=0.02):
     return norm
 
 
+def plot_image(image, extent, cmap="inferno", norm=None, ax=None, 
+               clab=r"Jy arcsec$^{-2}$",
+               xlab="RA offset [arcsec]",
+               ylab="Dec offset [arcsec]",
```

Comment:
Usually I use the \alpha and \delta Latex strings as in the tutorials. 

```
chan = 4
kw = {"origin": "lower", "interpolation": "none", "extent": imager.coords.img_ext}
fig, ax = plt.subplots(ncols=2, figsize=(6.0, 4))
ax[0].imshow(beam[chan], **kw)
ax[0].set_title("beam")
ax[1].imshow(img[chan], **kw)
ax[1].set_title("image")
for a in ax:
    a.set_xlabel(r"$\Delta \alpha \cos \delta$ [${}^{\prime\prime}$]")
    a.set_ylabel(r"$\Delta \delta$ [${}^{\prime\prime}$]")
fig.subplots_adjust(left=0.14, right=0.90, wspace=0.35, bottom=0.15, top=0.9)
```

For typical projections of square images the RA offset is technically \Delta \alpha \cos \delta, in units of arcseconds.